packages/core/src/modal/modal.element.ts

```diff
@@ -150,8 +150,9 @@ export class CdsModal extends CdsInternalOverlay {
     this.observers.forEach(o => o.disconnect());
   }
 
-  private setScrollableProperties() {
+  private async setScrollableProperties() {
     if (this.hidden === false) {
+      await this.updateComplete; // wait until after render to measure if scrollable
```

Comment:
There is chance in storybook that the scroll check fires before the first render causing an exception. This waits until render then checks which should be more accurate and removes the error. I'm not sure if there is a great way to test this since its a lifecycle issue.